CMSgov/mdct-mcr
# edit access measure details (last part of ModalDrawerReportPage)

## Description

Had to get the report drawer hydrating and saving data 👍 

Resolves: https://qmacbis.atlassian.net/browse/MDCT-1981

### How to test

1. Go Section C: Access Measures and put it through its paces. 
2. Add, edit measure, enter details, edit details, delete, etc. Do it in different orders with multiple measures, mixing up between measures even. Everything should work great. 

### Changed Dependencies

n/a

## Code author checklist

## Reviewer checklist (two different people)
